src/main/kotlin/com/mineinabyss/blocky/helpers/GenericHelpers.kt

```diff
@@ -53,11 +56,17 @@ const val stoneHitSound = "blocky.stone.hit"
 const val stoneStepSound = "blocky.stone.step"
 const val stoneFallSound = "blocky.stone.fall"
 
-fun breakBlockyBlock(block: Block, player: Player?) {
+fun attemptBreakBlockyBlock(block: Block, player: Player?) {
```

Comment:
This has been renamed from `breakBlockyBlock` to `attemptBreakBlockyBlock` which sounds like the block break will not happen under certain circumstances, but that is not what it looks like in this function

Reply:
it will whenever theres checks for WorldGuard and other protection plugins though not too important until breaking actually works